unity/Runtime/Components/Shapes/MjHeightFieldShape.cs

```diff
@@ -75,17 +80,17 @@ public class MjHeightFieldShape : IMjShape {
     var assetName = scene.GenerationContext.AddHeightFieldAsset(this);
 
     if (Application.isPlaying) {
-      scene.postInitEvent += (unused_first, unused_second) =>
```

Comment:
I had to add these names for our internal tools, please leave them as is.

Reply:
Ah, I thought it was my editor applying style, I'll swap it back.